lib/any_login/providers/clearance.rb

```diff
@@ -20,7 +20,3 @@ def any_login_sign_in
     end
   end
 end
-
-AnyLogin.provider = AnyLogin::Provider::Clearance
-
-AnyLogin::ApplicationController.send(:include, ::Clearance::Controller)
```

Comment:
This was moved to the anylogin application controller. Since this file is no longer required this statement is never executed.